src/test/utils/apiserver.ts

```diff
@@ -13,84 +13,90 @@ export class ApiServer {
   private static _hackbotPassword: string = 'password123456789';
   private static _adminUsername: string = 'admin_user123456789';
   private static _adminPassword: string = 'admin_pass123456789';
-  
+
   public static get Port(): number {
     return this._port;
   }
-  
+
   public static get PusherHost(): string {
     return this._pusherHost;
   }
-  
+
   public static get PusherKey(): string {
     return this._pusherKey;
   }
-  
+
   public static get PusherSecret(): string {
     return this._pusherSecret;
   }
-  
+
   public static get PusherPort(): number {
     return this._pusherPort;
   }
-  
+
   public static get PusherAppId(): string {
     return this._pusherAppId;
   }
-  
+
   public static get HackbotPassword(): string {
     return this._hackbotPassword;
   }
-  
+
   public static get AdminUsername(): string {
     return this._adminUsername;
   }
-  
+
   public static get AdminPassword(): string {
     return this._adminPassword;
   }
-  
+
   static start() {
-    
+
     return new Promise<void>((resolve, reject) => {
-      
+
+      let env: any = {
+        PORT: this._port,
+        HACKBOT_PASSWORD: this._hackbotPassword,
+        ADMIN_USERNAME: this._adminUsername,
+        ADMIN_PASSWORD: this._adminPassword,
+        PUSHER_URL: `http://${this._pusherKey}:${this._pusherSecret}@${this._pusherHost}:${this._pusherPort}/apps/${this._pusherAppId}`,
+      };
+
+      if ('MONGODB_URL' in process.env) {

```

Comment:
But now you've broken it on my machine. I actually need to set it since mongo isn't running on localhost. Maybe try changing the host your mongo runs on and pass the option in...